src/test/java/org/junitpioneer/jupiter/issue/IssueExtensionExecutionListenerTests.java

```diff
@@ -90,8 +89,8 @@ void abortedIssueTestCaseCreated() {
 		assertThat(allTests.size()).isEqualTo(1);
 
 		IssueTestSuite issueTestSuite = allTests.get(0);
-		assertAll(() -> assertThat(issueTestSuite.issueId()).isEqualTo("#123"),
```

Comment:
I would not remove the sense of `assertAll` as due the removal the tests behaves different. If you want to get rid of JUnit imports in favor for AssertJ then use `Soft Asserts` (blog example here: https://eliasnogueira.com/soft-asserts-why-should-you-use-them-for-unit-and-integration-tests/ )

Reply:
Why do we want soft assertions here, specifically? I do not see the added benefit...?

Reply:
> Why do we want soft assertions here, specifically? I do not see the added benefit...?

AssertAll (JUnit) or SoftAsserts (AssertsJ equivalent) perform all asserts before failing if one fails. Without it, the tests fails at the first failure and you don't get any information about the other asserts. So it's possible that you get a failure, fix it, run test again, get next failure etc. where you could get all failures in first run.

Reply:
@beatngu13 Please comment with your option and treat the PR based on that. If you agree with Mihaly then feel free to merge. Otherwise request changes :D

Reply:
I tend to lean towards no need for `SoftAssertions` in that case, but I have no strong opinion here.